broadbean/sequence.py

```diff
@@ -644,7 +662,7 @@ def element(self, pos):
         return elem
 
     @staticmethod
-    def _plotSummary(seq: Dict[int, Dict]) -> Dict[int, Dict[str, np.ndarray]]:
+    def _plot_summary(seq: Dict[int, Dict]) -> Dict[int, Dict[str, np.ndarray]]:
```

Comment:
perhaps worth creating a type alias for the broadbean Sequence dict ? `BBSequence = Dict[int, Dict]`. just for clarity of the code.
```suggestion
    def _plot_summary(seq: BBSequence) -> Dict[int, Dict[str, np.ndarray]]:
```

Reply:
definitely let's add this type alias here.